src/components/submission/metadata-row.vue

```diff
@@ -131,6 +133,9 @@ export default {
         __system.attachmentsPresent !== __system.attachmentsExpected;
     },
     editPath() {
+      if (this.form.dataExits && this.form.webFormsEnabled) {
```

Comment:
`webFormsEnabled` - should the "F" be capitalized? I see that it's not capitalized in other places.